box.py

```diff
@@ -39,10 +40,6 @@
 __author__ = 'Chris Griffith'
 __version__ = '3.0.1'
 
-ILLEGAL_ATTRIBUTES = ('if', 'elif', 'else', 'for', 'from', 'as', 'import',
```

Comment:
@cdgriffith `kwlist` is a superset of  `ILLEGAL_ATTRIBUTES` with these extra keywords:
```
{'False',
 'None',
 'True',
 'and',
 'assert',
 'break',
 'continue',
 'del',
 'finally',
 'global',
 'lambda',
 'nonlocal',
 'or',
 'pass',
 'try',
 'with'}
```